src/components/WebexMediaAccess/WebexMediaAccess.jsx

```diff
@@ -41,7 +41,7 @@ export default function WebexMediaAccess({
 
   return (
     <div className={cssClasses} style={style}>
-      <Icon name={screen.icon} />
+      <Icon name={screen.icon} size={48} />
```

Comment:
it's a bit strange to have 2 conflicting size values (36 and 48). I would prefer to use the name `camera_48' and map this in `Icon` from `camera_48` to `icon-camera_36` just for the legacy icons.